cpp/include/cugraph/visitors/graph_make_visitor.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include "erased_pack.hpp"
+#include "graph_envelope.hpp"
+#include "ret_terased.hpp"
+
+namespace cugraph {
+namespace visitors {
+// primary empty template:
+//
+template <typename vertex_t,
+          typename edge_t,
+          typename weight_t,
+          bool st,
+          bool mg,
+          typename Enable = void>
+struct graph_maker_visitor;
```

Comment:
This was done on purpose, because there's more functionality than just the visitor in there. The file name suggests content pertaining to the `visitor` design (verb = `make appropriate visitor` + related functionality) while the `visitor` name says: "I'm a visitor that makes graphs"...but in that same file there's more than that. The 2 names are too close, I agree, but I wanted to make that clear distinction. Can I please address that after merge (since every little change takes almost a whole day for CI to process)?